offer-manager-domain/src/main/groovy/life/qbic/business/offers/Offer.groovy

```diff
@@ -87,13 +87,19 @@ class Offer {
      */
     private double itemsWithoutOverheadNetPrice
 
-    /*
-     * Holds the determined overhead derived from the
+    /**
+     * Holds the determined overhead total derived from the
      * customer's affiliation.
      */
     private double overhead
 
-    /*
+    /**
+    * Holds the determined overhead ratio derived from the
+    * customer's affiliation.
+    */
+    private double overheadRatio
```

Comment:
is this the percentage of the overhead? If so maybe you could state that in the comment with an example to be able to differentiate between these new properties